deploy/example/resizing/README.md

```diff
@@ -0,0 +1,98 @@
+# Volume Resizing Example
+
+## Enable Volume Resizing Feature Gate
+
+In Kubernetes 1.14 and 1.15, CSI volume resizing is still alpha. Currently, Azure Disk CSI Driver just supports the **offline** scenario(Volume is currently not published or available on a node). So the following feature gate is needed to be enabled.
```

Comment:
make it more specific:
1. Kubernetes 1.14, 1.15: alpha
2. Kubernetes 1.16 beta
etc...

Reply:
done.